### DISCUSSION TITLE
customLocale property on bk-add-new-button is ignored

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#446] [Q&A] [answered]
### Which section?

Backoffice

### Which is the version?

back-kit@1.4.7, microlc-backend@0.8.1, microlc-frontend@0.9.0

### Description

I'm trying to use the `customLocale` property on a `bk-add-new-button` without success.
This is the code I'm using:
```json
 {
    "type": "element",
    "tag": "bk-add-new-button",
    "properties": {
        "customLocale": {
            "content": {
                "en": "En test",
                "it": "It test"
            }
        }
    }
}
```

### Actual Outcome

The text on the button is always `+ Add New`

### Expected Outcome

I'd like to have the button with the text `En test`

**Comment by akelity** [comment]
Hi @davidgarra 
I will check and either I or one of my colleagues will get back to you. :)

**Comment by clabene** [comment, marked as answer]
Hi @davidgarra,

`back-kit` components support `customLocale` property since property `1.4.14`. We recommend updating to the latest version, `1.5.0`.
[Here](https://docs.mia-platform.eu/docs/release-notes/v12.2.0#improvements-1) is the corresponding release note, while this is the [latest one](https://docs.mia-platform.eu/docs/release-notes/v12.3.1#back-kit-library).

Please note: the "+" sign in the `bk-add-new-button` component cannot be controlled through `customLocale` property, as it is not part of button label, but rather an icon. A `bk-button` component can be configured to emit an `add-new` event, just like the `bk-add-new-button`, while also allowing to configure the desired icon (including no icon). The second snippet from [this example](https://docs.mia-platform.eu/docs/microfrontend-composer/back-kit/components/button#example-basic-usage) represents such a `bk-button`.

**Reply by davidgarra** [reply]
thank you so much for the answer!!!

**Reply by clabene** [reply]
Thank you for reaching out!
